src/test/resources/org/prebid/server/it/openrtb2/sovrn/test-sovrn-bid-request.json

```diff
@@ -20,9 +20,7 @@
       "tagid": "tag_id",
       "ext": {
         "tid": "${json-unit.any-string}",
-        "bidder": {
-          "tagid": "tag_id"
-        }
```

Comment:
Was there a reason we removed this bidder/tagid value in this PR?

Reply:
I tried comparing it to Blackbird's openrtb-sovrn.proto: 

[](https://github.com/sovrn/blackbird/blob/23bc2e065166a5c705d9b1f7d47fc03dbb7b59d3/blackbird-openrtb-ext/src/main/proto/openrtb-sovrn.proto#L26C1-L34C1)

there is no bidder field handled there, so I thought it was redundant. Please correct me if I am wrong.